### ISSUE TITLE
Unable to downgrade version of Semgrep used in Github actions

### ISSUE THREAD

**Issue by harshshah2-crest** [returntocorp/sgrep-action#476]
- The latest released version of Semgrep 0.77.0 breaks our Github actions CICD pipeline as it errors out with [exit code 2](https://semgrep.dev/docs/cli-usage/#exit-codes).
- We tried to revert back to older Semgrep version v0.76.2 but we were unable to since each version is pushed to `v1` tag at dockerhub and in action.yml it always fetches the `v1` tag code.
ref: https://github.com/returntocorp/semgrep-action/blob/develop/action.yml#L6
- Due to this we are unable to use this action with different release versions.
- This issue is leading Semgrep stage to failure in our Github Actions CICD pipeline 

Github Action Job:
```
semgrep:
    runs-on: ubuntu-latest
    name: semgrep
    steps:
      - uses: actions/checkout@v2
      - name: Semgrep
        id: semgrep
        uses: returntocorp/semgrep-action@v1
        with:
          publishToken: ${{ secrets.SEMGREP_PUBLISH_TOKEN }}
```

**Comment by brendongo** [comment]
HI @harshshah2-crest sorry this fell through the cracks during the holidays. Are you able to provide more logs on this crash?

**Comment by ievans** [comment]
The need for ability to revision is addressed with the 0.86.5 release of Semgrep which moved action functionality into the regular CLI. Closing--@harshshah2-crest feel free to re-open if the problem re-occurs on 0.86.5!
